types/string-placeholder/string-placeholder-tests.ts

```diff
@@ -0,0 +1,19 @@
+import template from 'string-placeholder';
+
+// $ExpectType string
+template('{0} test', ['here']);
+
```

Comment:
Please add a case with an object.

```suggestion

template('${foo} test ${bar}', {foo: 'Bob', bar: '65'});

```